package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@basketry/react-query",
-  "version": "0.0.0",
+  "version": "0.2.0",
```

Comment:
We don't need to do this. Running the `version` GHA workflow will set the version correctly.